app/controllers/api/verify/document_capture_controller.rb

```diff
@@ -0,0 +1,79 @@
+module Api
+  module Verify
+    class DocumentCaptureController < BaseController
+      self.required_step = 'document_capture'
+      include ApplicationHelper
+
+      def create
+        result = @form ||= Idv::ApiDocumentVerificationForm.new(
+          verify_params,
+          liveness_checking_enabled: liveness_checking_enabled?,
+          analytics: analytics,
+        ).submit
+
+        analytics.idv_doc_auth_submitted_image_upload_form(**result.to_h)
+        if result.success?
+          enqueue_job
+
+          render json: {
+            status: 'in_progress',
+          }, status: :ok
+        else
+          render json: { error: result.errors }, status: :bad_request
```

Comment:
QQ: does the frontend always look for error (singular)? Looks like we're passing back an array or a hash, so I feel like changing the key would be clearer:
```suggestion
          render json: { errors: result.errors }, status: :bad_request
```

Reply:
I don't have a particular preference on singular or plural, but couple notes:

1. We want this to be consistent across the board with all FSMv2 APIs, where we're otherwise using `error` singular
2. Regardless what the client is _currently_ doing, we anticipated needing to make some revisions to adjust to the new API response structure regardless, so I wouldn't expect that would need to drive our decision here
3. In my local branch version of the "status" complement of this pull request, I an idea for a `def render_errors` method in `BaseController` to help guarantee consistency of these kinds of names / structure. 

```rb
      def render_errors(errors, status: :bad_request)
        raise ArgumentError, 'Errors should be given as a hash' if !errors.is_a?(Hash)
        if errors.values.any? { |value| !value.is_a?(Array) }
          raise ArgumentError, 'Error values should be an array of strings'
        end

        render json: { error: errors }, status: status
      end
```

Reply:
Yeah maybe we standardize on `errors` always being plural and always being an array, I would love that

Reply:
does that mean you want it to be 
`def render_errors(errors, status: :bad_request)
        raise ArgumentError, 'Errors should be given as a hash' if !errors.is_a?(Hash)
        if errors.values.any? { |value| !value.is_a?(Array) }
          raise ArgumentError, 'Error values should be an array of strings'
        end

        render json: { errors: [errors] }, status: status
      end`

Reply:
Maybe more like...
```
def render_errors(error_or_errors, status: :bad_request)
  render json: { errors: Array(error_or_errors) }, status: status
end
```


taking advantage of how `Array()` will leave arrays as-is, but turn single elements into arrays:
```ruby
Array(1)
=> [1]
Array([1, 2])
=> [1, 2]
```
we might want to add in a check/guard for hashes, because this is undesirable behavior IMO
```ruby
Array({ a: 1, b: 2 })
=> [[:a, 1], [:b, 2]]
```